src/org/labkey/test/pages/core/login/DatabaseAuthConfigureDialog.java

```diff
@@ -0,0 +1,165 @@
+package org.labkey.test.pages.core.login;
+
+import org.labkey.test.LabKeySiteWrapper;
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.components.html.OptionSelect;
+import org.labkey.test.util.LogMethod;
+import org.labkey.test.util.LoggedParam;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class DatabaseAuthConfigureDialog extends AuthDialogBase<DatabaseAuthConfigureDialog>
+{
+
+    public DatabaseAuthConfigureDialog(LoginConfigRow row)
+    {
+        super(getFinder("Configure Database Authentication", row.getDriver()));
+    }
+
+    private static ModalDialogFinder getFinder(String title, WebDriver driver)
+    {
+        return new ModalDialogFinder(driver).withTitle(title);
+    }
+
+    // get password strength
+    public PasswordStrength getPasswordStrength()
+    {
+        if (elementCache().weakButton.getAttribute("class").contains("active"))
+            return PasswordStrength.Weak;
+        else return PasswordStrength.Strong;
+    }
+
+    public DatabaseAuthConfigureDialog setPasswordStrength(PasswordStrength desiredStrength)
+    {
+        if (getPasswordStrength().equals(desiredStrength))
+            return this;
+        else
+            if (desiredStrength.equals(PasswordStrength.Strong))
+                elementCache().strongButton.click();
+            else
+                elementCache().weakButton.click();
```

Comment:
This `if/else` is hard to read without brackets. Removing the nesting makes it more readable to me.
```
if (getPasswordStrength().equals(desiredStrength))
    return this;
else if (desiredStrength.equals(PasswordStrength.Strong))
    elementCache().strongButton.click();
else
    elementCache().weakButton.click();
```